wgengine/netstack/netstack.go

```diff
@@ -215,6 +215,8 @@ func Create(logf logger.Logf, tundev *tstun.Wrapper, e wgengine.Engine, mc *magi
 	}
 	ns.ctx, ns.ctxCancel = context.WithCancel(context.Background())
 	ns.atomicIsLocalIPFunc.Store(tsaddr.NewContainsIPFunc(nil))
+	ns.tundev.PostFilterPacketInboundFromWireGaurd = ns.injectInbound
```

Comment:
I think I've convinced myself these values don't depend on the stuff before their old location (notwithstanding the goroutine). 👀